sda-commons-server-kafka/src/main/java/org/sdase/commons/server/kafka/prometheus/KafkaConsumerMetrics.java

```diff
@@ -15,16 +15,21 @@
 import org.slf4j.LoggerFactory;
 
 /**
- * Custom Prometheus Collector scraping Kafka Metrics for all registered message listeners.<br>
- * Specify the metrics to be handled by Prometheus in {@link #KAFKA_METRICS}.
+ * @deprecated use micrometer library to expose Kafka metrics see {@link
+ *     org.sdase.commons.server.kafka.MicrometerConsumerListener}. Custom Prometheus Collector
+ *     scraping Kafka Metrics for all registered message listeners.<br>
+ *     Specify the metrics to be handled by Prometheus in {@link #KAFKA_METRICS}.
  */
+@Deprecated(since = "5.8.1", forRemoval = true)
 public class KafkaConsumerMetrics extends Collector {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(KafkaConsumerMetrics.class);
 
   /** The prefix for the metrics name as they are published to Prometheus */
   private static final String METRIC_NAME_PREFIX = "kafka_consumer_";
 
+  private static final String METRIC_DESCRIPTION_PREFIX = "**DEPRECATED** ";
```

Comment:
Is it worth to extract the value as a constant? I would prefer to name it "DEPRECATION_PREFIX" if you want to keep it.

Reply:
removed constant